internal/transport/http2_client.go

```diff
@@ -59,6 +59,8 @@ import (
 // atomically.
 var clientConnectionCounter uint64
 
+var GoAwayLoopyWriterTimeout = time.Second
```

Comment:
Why is this exported?

Reply:
Did some discussion happen on the value for this timeout? `1s` seems quite short to me. If a discussion happened, it would be worth adding a comment here to say why `1s` is considered a reasonable value for this timeout?

Reply:
@easwars Updated this time out to 5s as per discussion with Doug.